README.md

```diff
@@ -55,6 +56,30 @@ Just stick this in your `<head>`:
 
 A **preview** of the different themes is available [on the **demo page**](https://watercss.kognise.dev/#installation)! ⚡
 
+#### Installing from NPM
+
+Alternatively, to install Water.css through a package manager, you can run:
+
+```
+# Using npm
+npm install water.css
+
+# Using yarn
+yarn add water.css
+```
+
+<details>
+  <summary><strong>In JavaScript (where CSS imports are supported)</strong></summary>
+
+  <pre><code>import 'water.css/out/water.css'</code></pre>
+</details>
+
+<details>
+  <summary><strong>In CSS or similar</strong></summary>
+
+  <pre><code>@import './node_modules/water.css/out/water.css';</code></pre>
```

Comment:
I'm not so sure if these instructions are a good idea tbh.
How to import CSS from `node_modules` highly depends on the build setup that you use, whether that's Parcel, Webpack (or rather: the webpack loader), PostCSS...

- `url()` yes or no?
- assume module resolution? (`"water.css/out/..."`)
- module resolution but resolve from project root? (the webpack way - `"~water.css/out/..."`)
- no module resolution? (`"./node_modules/water.css/"`)
- why the relative import? (instead of `"/node_modules/water.css/out/..."`)

We either need to specify which setup our instructions are meant for or write more generic instructions, like "if your setup allows importing CSS from `node_modules`, you can import it from `water.css/out/water.css`" or something along these lines.

Might want to check out how other CSS libraries do this? 🤔

Reply:
Yeah this stuff is pure magic to me, to be honest. We might just want to pick one example and let people extrapolate from there? Eg. Create-React-App or Next.js or something

Reply:
Yeah fair. I know the direct CSS import in JS works basically everywhere but the import from CSS is a bit iffy.